src/SFA.DAS.Assessor.Functions/Functions/ExternalApiDataSync/ExternalApiDataSyncFunction.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+using SFA.DAS.Assessor.Functions.Domain.ExternalApiDataSync.Interfaces;
+
+namespace SFA.DAS.Assessor.Functions.Functions.ExternalApiDataSync
+{
+    public class ExternalApiDataSyncFunction
+    {
+        private readonly IExternalApiDataSyncCommand _command;
+
+        public ExternalApiDataSyncFunction(IExternalApiDataSyncCommand command)
+        {
+            _command = command;
+        }
+
+        [FunctionName("ExternalApiDataSyncFunction")]
+        public async Task Run([TimerTrigger("%ExternalApiDataSyncFunctionSchedule%", RunOnStartup = true)]TimerInfo myTimer, ILogger log, ExecutionContext context)
```

Comment:
This "ExternalApiDataSyncFunctionSchedule" is defined in another file with an actual schedule - which you have from the original code - you can't see that when the project is load - swap to directory view

Reply:
Don't forget the other files for this schedule need to be amended to include the default schedule